app/pages/EditBreakingNewsPage.scss

```diff
@@ -47,3 +54,26 @@
 .labelSubtext {
   color: $color-grey5;
 }
+
+.breaking-news-round-edges {
```

Comment:
Nit: Could we rename this to something like `.breaking-news-status` or `.breaking-news-status-badge`? "round-edges" makes it a little hard to remember exactly what this CSS class is used in, and it's not really a generic "utility" class because it's doing more than just rounding the edges.